independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/base/src/main/docker/Dockerfile.tpl.qute.legacy-jar

```diff
@@ -1,12 +1,7 @@
 {#include Dockerfile-layout type='legacy-jar'}
     {#quarkusbuild}{buildtool.cli} {buildtool.cmd.package-legacy-jar}{/quarkusbuild}
-    {#image}{dockerfile.legacy-jar.from} {/image}
-    {#args}
-ARG JAVA_PACKAGE={dockerfile.legacy-jar.java-package}
-ARG RUN_JAVA_VERSION={dockerfile.legacy-jar.run-java-version}
-    {/args}
     {#copy}
 COPY {buildtool.build-dir}/lib/* /deployments/lib/
-COPY {buildtool.build-dir}/*-runner.jar /deployments/app.jar
+COPY {buildtool.build-dir}/*-runner.jar /deployments/quarkus-run.jar
```

Comment:
Is this change necessary or just preferred?

Reply:
I believe it doesn't matter it's just a matter of the jar's naming.

@ia3andy, do you have any consideration?

Reply:
Yes this is needed by the common entrypoint. The alternative would be to pass a parameter to deal with names in the entrypoint (but that's more complex to achieve and less clean IMO):
```
ENTRYPOINT [ "java", "-jar", "/deployments/quarkus-run.jar" ]
```